src/ui.data-list/UDataList.vue

```diff
@@ -39,24 +39,15 @@
               />
             </slot>
 
-            <template v-if="element.isActive">
-              <div v-bind="labelCrossedAttrs">
-                <!--
-                  @slot Use it to modify label.
-                  @binding {object} item
-                -->
-                <slot name="label" :item="element">
-                  {{ element[labelKey] }}
-                </slot>
-              </div>
-            </template>
-            <template v-else>
-              <div v-bind="labelAttrs">
-                <slot name="label" :item="element">
-                  {{ element[labelKey] }}
-                </slot>
-              </div>
-            </template>
+            <div v-bind="isActive(element) ? labelCrossedAttrs : labelAttrs">
```

Comment:
Are you shure you need to add crosses styles is element is active? I don't think so.